src/Api/Controllers/UsersInviteController.php

```diff
@@ -168,7 +169,17 @@ public function processUserInvite(string $hash) : Response
             //Check if user already exists
             $userExists = Users::getByEmail($usersInvite->email);
             $newUser = $userExists;
-            $this->userData->getDefaultCompany()->associate($userExists, $this->userData->getDefaultCompany());
+
+            $userAssociated = UsersAssociatedApps::findFirst([
```

Comment:
@tiberius19 we need to also change the invitation / generate a new invitation ID, maybe its best to just allow use to have a history of the invites instead of recycling , so lets leave it deleted and add a new record

Reply:
Got it